tinker/templates/admin/redirects/home.html

```diff
@@ -424,6 +438,13 @@ <h4 class="modal-title" id="myModalLabel">Redirect Information</h4>
                                 <input type="text" id="modal-last-edited-edit" class="uneditable edit" readonly/>
                             </td>
                         </tr>
+                        <tr>
+                            <th>Notes:</th>
+                            <td>
+                                <span id="modal-edit-notes-display" class="display">No notes were loaded</span>
```

Comment:
why do you display "No notes were loaded"?

Reply:
I initially added it in since all other fields displayed on that modal also have that in case there is no data to be loaded, but it looks like if no data is loaded for the notes field then "None" is displayed so I removed the span.

Reply:
The schema would have to be updated on both xp and prod for this change.

Reply:
👍 

As far as updating the schema, can you send the sql to update the app.db file?

Reply:
I ran this sql locally on my sqlite3 db `ALTER TABLE bethel_redirect ADD notes VARCHAR(256);`

Reply:
👍 thanks!

Reply:
It turns out that the span above that displays "No notes were loaded" was the span that the note gets loaded into so I re-added that.